pages/glossary/essential-text-presentation.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Essential text presentation
+key: essential-text-presentation
+unambiguous: true
+objective: false
+---
+
+The presentation of text is considered essential in one of more of the following scenarios:
+
+1. The text and its presentation is part of a brand; or
+2. The text is part of a digitized image of a physical object; or
+3. The text is part of a free-form digital illustration; or
+3. The text is written, drawn, or otherwise hand-crafted; or
+4. The text is part of an image with other graphical objects, where its relationship is informative; or
+5. Changing part of the presentation would alter the meaning of content on the page.
+
+Examples of text for which the presentation is essential include:
+
+1. Logos, product names, or slogans
+2. Image of a hand-written letter, picture of a street sign, or a scanned contract
```

Comment:
Would be good in my opinion to keep consistency between scenarios above and examples, reflecting with examples the numbers in the ordered list of scenarios.

```suggestion
2. A photograph of a product containing text
3. Image of a hand-written letter, picture of a street sign, or a scanned contract
```